test/functional/mempool_compatibility.py

```diff
@@ -33,13 +38,15 @@ def setup_network(self):
             None,
         ])
         self.start_nodes()
-        self.import_deterministic_coinbase_privkeys()
+        self.init_wallet(0)  # only import_deterministic_coinbase_privkeys for old_node
```

Comment:
Does this line need to change? If yes, why?

Reply:
Good point - this change is not necessary. I got rid of it and force pushed